src/test/scala/cromwell/binding/RuntimeAttributeSpec.scala

```diff
@@ -85,4 +111,11 @@ class RuntimeAttributeSpec extends FlatSpec with Matchers {
   "WDL file without runtime" should "not have imported runtime information" in {
     assert(NamespaceWithoutRuntime.workflows.head.calls.forall {_.task.runtimeAttributes.isEmpty})
   }
+
+  "WDL file with failOnStderr runtime" should "identify failOnStderr for appropriate tasks" in {
```

Comment:
This could be one test.  Also it might be nice to match the calls extracted from the workflow to the expected call names, it seems a bit opaque that calls are in this list in the order of their definition in the workflow.